src/main/resources/standardTestCases/scatter_chain/scatter_chain.wdl

```diff
@@ -0,0 +1,53 @@
+workflow scatter_chain {
```

Comment:
It would be nice in general for Centaur tests to describe the condition being tested.  Having been out this week, I have no idea what this test is about. 😦